src/Collectors/DBQueryCollector.php

```diff
@@ -61,4 +57,23 @@ public static function getName(): string
     {
         return 'query-collector';
     }
+
+    private function getQueryName(string $sql): string
+    {
+        $fallback = 'Eloquent Query';
+
+        try {
+            $query_type = QuerySplitter::getQueryType($sql);
+            $tables = QuerySplitter::splitTables($sql);
+
+            if (isset($query_type) && is_array($tables)) {
+                // Query type and tables
+                return $query_type . ' ' . join(', ', array_values($tables));
+            }
+
+            return $fallback;
+        } catch (Exception $e) {
+            return $fallback;
```

Comment:
You could also keep the catch block empty and move the `return $fallback` after the whole try-catch block to same 1 line of code (#crazy-gains).

Reply:
I never liked empty `catch` blocks 😅

Reply:
Fair enough.